changelog.json

```
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "3.1.0": [
+      "[Added] You can diff a consecutive range of commits - #14682",
+      "[Added] Submodule changes now displays an explanatory, interactive diff screen  - #15176",
```

```suggestion
      "[Added] Submodule changes now display an explanatory, interactive diff screen  - #15176",
```

Sorry.. my fault bad grammar suggestion..